app/views/users/show.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Display Single User</h1>
+<h1 class="text-seondary mb-5">Single user</h1>
+
+<div class="d-flex align-items-center mb-3 gap-3 w-100">
+  <div>
+    <img src="<%= @user.photo %>" alt="user photo" class= "border border-dark border-5 p-5">
+  </div>
+  <div class=" border border-dark border-5 w-100 p-3">
+    <h2 class="flex-grow-1 p-2"> <%= @user.name %></h2>
+    <div class="d-flex justify-content-end">
+      <p>Number of posts: <%= @user.posts_counter %></p>
```

Comment:
- [ ] Again here kindly style your view for each user to look like the given wireframe

Given design
![image](https://user-images.githubusercontent.com/81932478/230741424-befdcb60-2d0a-43e8-b657-88522c7ae436.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/230741443-d444b5f9-a3af-4035-b7f0-3e8fc61d97cc.png)